com.github.petebuffon.yafc.appdata.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.github.petebuffon.yafc</id>
+  <name>YAFC</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <summary>YAFC is a Factorio calculator, planner and analyser. The main goal of YAFC is to cope with heavily modded games.</summary>
+  <description><p>YAFC is more than just calculator. It contains multiple analysis algorithms and tries to understand what is going on in your modpack. These analyses go as deep as trying to estimate amount of ores generated by the map generator, and as wide as calculating late-game base trying to figure how much of everything you are going to need. It knows which items are important and which are less important, which recipes are efficient and which are less efficient.</p><p>I have started working with YAFC when other tools (such as Helmod) failed to work with deeply recursive Pyanodon recipes. YAFC handles recursive recipes extremely well, it uses Google OrTools as its model solver</p><p>YAFC also includes several tools, the most important currently is "Never Enough Items" - it is FNEI on steroids, showing not only recipes to produce or consume an item, but also which recipes you probably want to use, and how much.</p><p>NOTE: This wrapper is not verified by, affiliated with, or supported by Mojang or Microsoft.</p></description>
+  <url type="homepage">https://github.com/ShadowTheAge/yafc</url>
+  <screenshots>
+    <screenshot type="default">https://github.com/ShadowTheAge/yafc/blob/master/Docs/Media/Main.gif</screenshot>
+  </screenshots>
+  <launchable type="desktop-id">com.github.petebuffon.yafc.desktop</launchable>
+  <project_license>GPL-3.0-only</project_license>
+  <developer_name>ShadowTheAge</developer_name>
+  <releases>
+    <release version="0.5.8" date="2021-07-25"/>
+  </releases>
+  <content_rating type="oars-1.0">
```

Comment:
`none` can be skipped in `oars-1.1`.

Reply:
Changed to <content_rating type="oars-1.1"/>